src/urbit/app/chess.hoon

```diff
@@ -1228,13 +1197,13 @@
       :-  `[updated-game u.new-position new-fen-repetition special-draw-available |4.game-state]
       ?.  special-draw-claim
         :~  position-update-card
+            :*  %give  %fact  ~[/archive]
+                %chess-game  !>(game.game-state)
+            ==
```

Comment:
I attempted to find every instance where a game would end and gave a fact to the frontend subscription for `/archive`s.

Reply:
let me know if I missed one.